src/components/SideStepper.tsx

```diff
@@ -57,7 +55,7 @@ const SideStepper: React.FC<SideStepperProps> = ({ jumpToTopic, activeTopic }) =
                         <Typography variant="caption">Extremes</Typography>
                     </StepButton>
                     <StepContent>
-                        <Typography> Fourth topic bla </Typography>
+                        <Button variant="contained" onClick={() => navigate('/app/extremes')}>Open App</Button>
```

Comment:
The path `/app/extremes` does not yet exist. We should disable this button as well, until the route is added.

Reply:
Alright